tests/format/flow/type-alias/jsfmt.spec.js

```diff
@@ -0,0 +1 @@
+run_spec(__dirname, ["flow", "babel"]);
```

Comment:
Let's run flow test with `babel-flow` instead of `babel`.

Reply:
df383c7